src/components/PreviewResults.tsx

```diff
@@ -0,0 +1,51 @@
+import { FeatureCollection } from "geojson";
+import PropTypes from "prop-types";
+import React, { useEffect } from "react";
+import styled from "styled-components";
+
+const Preview = styled.div`
+  border: 1px solid lightgrey;
+  padding: 5px;
+  margin: 0 5px 0 5px;
```

Comment:
This is the same as using:
```suggestion
  margin: 0 5px;
```